pkg/service/webhook/model.go

```diff
@@ -0,0 +1,38 @@
+package webhook
+
+type Noun string
+type Action string
+
+type Webhook struct {
```

Comment:
I'm not sure what Noun & Action represent. Mind adding docs?

Reply:
Added comments and a few more functions that explain the functionality of this